distributed/deploy/cluster.py

```diff
@@ -50,12 +51,14 @@ class Cluster:
 
     _supports_scaling = True
 
-    def __init__(self, asynchronous):
+    def __init__(self, asynchronous, quiet=False):
         self.scheduler_info = {"workers": {}}
         self.periodic_callbacks = {}
         self._asynchronous = asynchronous
         self._watch_worker_status_comm = None
         self._watch_worker_status_task = None
+        self._cm_logs = []
```

Comment:
Curious what "cm" stands for? Oh, I guess cluster manager logs? Maybe include a little comment above saying what the name is, or give a clearer name?

Reply:
Yeah it was cluster manager, perhaps `_cluster_manager_logs` or just `_cluster_logs` would be better.